colossalai/inference/hybridengine/polices/llama.py

```diff
@@ -45,8 +45,10 @@ def __init__(self) -> None:
 
     def module_policy(self):
         policy = super().module_policy()
-
-        if self.shard_config.inference_gptq:
+        if (
+            "inference_gptq" in self.shard_config.extra_kwargs.keys()
```

Comment:
You can use `self.shard_config.extra_kwargs.get("inference_gptq", False)`

Reply:
Fixed.